.pipelines/singletenancy/azure-cni-overlay/azure-cni-overlay-e2e-step-template.yaml

```diff
@@ -161,3 +161,87 @@ steps:
     name: "Cleanupartifactdir"
     displayName: "Cleanup artifact dir"
     condition: always()
+
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: $(AZURE_TEST_AGENT_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        make -C ./hack/aks windows-nodepool-up AZCLI=az SUB=$(SUB_AZURE_NETWORK_AGENT_BUILD_VALIDATIONS) CLUSTER=${{ parameters.clusterName }}
+        echo "Windows node are successfully added to v4 Overlay Cluster"
+        kubectl cluster-info
+        kubectl get node
+        kubectl get po -owide -A
+    name: "Add_Windows_Node"
+    displayName: "Add windows node on v4 overlay cluster"
+
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: $(AZURE_TEST_AGENT_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        set -e
+        echo "Deploying windows CNS on v4 overlay cluster"
+        export WINDOWS_CNS_VERSION=$(make cns-version)
+        envsubst '${WINDOWS_CNS_VERSION}' < test/integration/manifests/cns/daemonset-windows.yaml | kubectl apply -f -
+        echo "wait for CNS pods deployment ready"
+        kubectl rollout status ds azure-cns-win -n kube-system --timeout 5m
+        kubectl get po -owide -A
+        kubectl describe ds azure-cns-win -n kube-system | grep Image
```

Comment:
Why do we need this ?

Reply:
it will print which CNS image we are using:
paulyu@paul-microsoft:/mnt/c/Users/paulyu$ kubectl describe ds azure-cns-win -n kube-system | grep Image
    Image:       mcr.microsoft.com/containernetworking/azure-cns:v1.5.11